crates/sp-domains-fraud-proof/src/verification.rs

```diff
@@ -138,18 +151,29 @@ where
     let mut ordered_extrinsics =
         deduplicate_and_shuffle_extrinsics(bundle_extrinsics_digests, shuffling_seed);
 
-    // NOTE: the order of the inherent extrinsic MUST aligned with the
-    // pallets order defined in `construct_runtime` macro for domains.
-    // currently this is the following order
+    // NOTE: the order of the inherent extrinsics MUST be the same as the pallet order defined in
+    // the `construct_runtime` macro for domains.
+    // Currently this is the following order:
     // - timestamp extrinsic
     // - executive set_code extrinsic
     // - messenger update_domain_allowlist extrinsic
     // - block_fees transaction_byte_fee_extrinsic
     // - domain_sudo extrinsic
-    // since we use `push_front` the extrinsic should be pushed in reversed order
+    // - evm_tracker contract_creation_allowed_by extrinsic
```

Comment:
the order is incorrect here. EVM tracker as per the `contruct_runtime` of the EVM is after messenger

https://github.com/autonomys/subspace/blob/main/domains/runtime/evm/src/lib.rs#L840